README.md

```
@@ -3,8 +3,8 @@
 </p>
 
 <p align="center">
-  <a href="https://www.npmjs.org/package/element-plus">
-    <img src="https://img.shields.io/npm/v/element-plus.svg" />
+  <a href="https://npmx.dev/element-plus">
+    <img src="https://npmx.dev/api/registry/badge/element-plus" />
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)